docs/GettingStarted.md

```diff
@@ -1618,9 +1620,9 @@ Datadog.configure do |c|
 end
 ```
 
-This enables you to set this value on a per tracer basis, so you can have for example several applications reporting for different environments on the same host.
+This enables you to set this value on a per application basis, so you can have for example several applications reporting for different environments on the same host.
 
-Ultimately, tags can be set per span, but `env` should typically be the same for all spans belonging to a given trace.
+Tags can also be set directly on individual spans, which will override any conflicting tags defined at the application level.
```

Comment:
```suggestion
Tags can also be set directly on individual spans, which will superseed any tags defined at the application level.
```

Reply:
I like this clarification, but I believe it has a typo. I'll adopt it sans the typo!